{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/sqlalchemy_models/__init__.py

```diff
@@ -57,7 +57,6 @@ def includeme(config):
 
     """
     settings = config.get_settings()
-    settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
```

Comment:
do not remove it from the alchemy version, only zodb

Reply:
oops. fixed.